cmd/kflex/main.go

```diff
@@ -201,6 +231,17 @@ func init() {
 	createCmd.Flags().BoolVarP(&chattyStatus, "chatty-status", "s", true, "chatty status indicator")
 	createCmd.Flags().StringArrayVarP(&hookVars, "set", "e", []string{}, "set post create hook variables, in the form name=value ")
 
+	adoptCmd.Flags().StringVarP(&kubeconfig, "kubeconfig", "k", "", "path to kubeconfig file")
```

Comment:
Is there a default to the KUBECONFIG environment variable with standard semantics? Readers should probably be told the answer. And the answer should probably be "yes".